### ISSUE TITLE
Rates, how to set them in the docker-compose.yml file

### ISSUE THREAD

**Issue by Solamain** [r15ch13/arkcluster#13]
Yeah, titles kinda says it all. lol

**Comment by aciddensity** [comment]
@Solamain, I don't believe there is a way to set rates in the docker-compose file directly. Instead you can set them in your GameUserSettings.ini file, which you can specify in your docker-compose file by changing the value of GAME_USERSETTINGS_INI_PATH. Keep in mind the path of this file will be in the perspective of the container and not the host system.
